exe/cli/cmd/ffs_getcidconfig.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	ffsGetCidConfigCmd.Flags().StringP("token", "t", "", "FFS auth token")
+
+	ffsCmd.AddCommand(ffsGetCidConfigCmd)
+}
+
+var ffsGetCidConfigCmd = &cobra.Command{
+	Use:   "getCidConfig [cid]",
+	Short: "Retuns the config for the provided cid",
+	Long:  `Retuns the config for the provided cid`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		err := viper.BindPFlags(cmd.Flags())
+		checkErr(err)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*60)
+		defer cancel()
+
+		if len(args) != 1 {
+			Fatal(errors.New("you must provide cid argument"))
+		}
+
+		c, err := cid.Parse(args[0])
+		checkErr(err)
+
+		s := spin.New("%s Getting defautlt cid config...")
```

Comment:
Typo `defautlt`

Reply:
Maybe always using capital `C` or using always lowercase `c` might be a good rule?

Reply:
For "cid" or "Cid"?

Reply:
No strong preferences, both LGTM. Flipping a coin: `Cid`.

Reply:
Hmm if it's just a coin toss, I might go with lower case for now. I think it mostly (or all) lower case already, and as I started switching to "Cid" I saw all these issues with it seeming confusing if it was the first work in a sentence where other sentences in the same context aren't capitalized, also the discrepancy between flag names like `cid` and docs saying things like "use the Cid flag". Just easier to leave as "cid" for now.

Reply:
SGTM.